roles/ffh.dhcp_server/tasks/main.yml

```diff
@@ -1,25 +1,23 @@
 ---
-- name: Install isc-dhcp-server
-  apt: name=isc-dhcp-server update_cache=yes
 
-- name: Install dhcpd-pools
-  apt: name=dhcpd-pools
+- name: Install kea-dhcp4-server
+  apt: name=kea-dhcp4-server update_cache=yes
   
-- name: Ramdisk in fstab eintragen
-  notify: Restart isc-dhcp-server
-  mount: name=/var/lib/dhcp src=tmpfs fstype=tmpfs opts='defaults,size=100M' state=mounted
+- name: Add a ramdisk entry in fstab
```

Comment:
Although frequently rebooting a supernode is not recommened, we are losing dhcp state, if we put everything on a ramdisk. If we use ramdisks we have to make sure, that lease candidate addresses are probed properly.

Not using a ramdisk has privacy implications.